### ISSUE TITLE
关于Laplacian aleatoric uncertainty loss的一些问题

### ISSUE THREAD

**Issue by 2anchao** [Xianpeng919/MonoCon#25]
Laplacian aleatoric uncertainty loss在论文中描述如下：
<img width="356" alt="企业微信截图_1672992075652" src="https://user-images.githubusercontent.com/37031491/210956865-b0ba4834-43b0-4248-a2a9-46b9b0317f64.png">
代码中的写法：
loss = 1.4142 * torch.exp(-log_variance) * torch.abs(input - target) + log_variance
自己认为对齐的写法：
loss = 1.4142 * log_variance**-1 * torch.abs(input - target) + totch.log(log_variance)

包括在推理中的sigma值对score进行refine，我认为您使用torch.exp(log_variance)代替了原来的log_variance，将不确定性分布指数化，使其值域>0, 感觉这种做法有益，但具体说不上来，请帮忙解答一下。
